File: euler56.jl
max = 0

for a = big(99:-1:1), b = big(99:-1:1)
	numstr = "$(a^b)"
	val = 0
	for c in numstr
		val += (int(c)-48)
		if val > max
			max = val
			println(max)
		end
	end	
end

File: euler56.py
max = 0

for a in range(99,1,-1):
	for b in range(99,1,-1):
		numstr = str(a**b)
		val = 0
		for c in numstr:
			val += int(c)
			if val > max:
				max = val
				print max

